### PR TITLE
fixed styling

### LINKED ISSUE #47: Make style consistent

### PR DESCRIPTION
**Pull request by andrewlieu**
fixed #47

### CODE REVIEW

File: client/.eslintrc.json [hunk 1]
```diff
@@ -1,27 +0,0 @@
-{
```
Comment (bchee) [inline client/.eslintrc.json]: Why did you delete this

File: client/.eslintignore [hunk 2]
```diff
@@ -1,3 +0,0 @@
-**/build/**
```
Comment (bchee) [inline client/.eslintignore]: Why did you delete this too

File: client/src/components/Survey/Star/Star.css [hunk 3]
```diff
@@ -102,11 +102,9 @@ input:checked ~ .ratingControl-stars:active {
   font-family: 'Audiowide';
   font-style: normal;
   font-weight: 400;
-  src:
-    local('Audiowide'),
-    local('Audiowide-Regular'),
+  src: local('Audiowide'), local('Audiowide-Regular'),
     url(http://themes.googleusercontent.com/static/fonts/audiowide/v2/8XtYtNKEyyZh481XVWfVOj8E0i7KZn-EPnyo3HZu7kw.woff)
```
Comment (bchee) [inline client/src/components/Survey/Star/Star.css]: This should be https

File: client/src/components/Survey/Survey.jsx [hunk 4]
```diff
@@ -66,12 +66,14 @@ function displayQuestions(blocks) {
 class Survey extends Component {
   render() {
     return (
-      <section className="Survey">
```
Comment (bchee) [inline client/src/components/Survey/Survey.jsx]: Section tag was correct here

**Comment by bchee** [comment]
You delete all the eslint configuration files. Travis is failing because of that.

File: client/.eslintignore (hunk 2 above)
Comment (partyshah) [reply client/.eslintignore]: we were playing around last night. eslint was doing weird windows stuff with linebreaks. @andrewlieu revert back.
